src/common/layouts/Header/Header.jsx

```diff
@@ -65,10 +73,12 @@ const Header = props => (
 
 Header.propTypes = {
   shouldDisplaySearchBox: PropTypes.bool.isRequired,
+  shouldDisplayHoldingPen: PropTypes.bool.isRequired,
 };
 
 const stateToProps = state => ({
   shouldDisplaySearchBox: state.router.location.pathname !== '/',
+  shouldDisplayHoldingPen: !Set(state.user.getIn(['data', 'roles'])).has('betauser')
```

Comment:
Logic should be, if userRoles has `superuser` or `cataloger`.

example: https://github.com/inspirehep/inspirehep-ui/blob/master/src/common/containers/EditRecordActionContainer.jsx

Reply:
I see that we'll have some duplicate constants but it's OK for now, we can refactor later.